### PR TITLE
Submit 2020-07-27 meeting notes

### PR DESCRIPTION
**Pull request by mdelasa**
Signed-off-by: Martin De Lasa <mdelasa@github.com>

### CODE REVIEW

**Comment by jfpanisset** [comment]
Looks good, would approve if I had write access.

**Review by AWhetter — APPROVED** [review]
Looks good to me too. I'll wait a few days to merge in case anyone else has comments.

**Pull request merged**
